### ISSUE TITLE
Add New command does NOT clear the Data Entry forms in a Climsoftv4 client-server environment.

### ISSUE THREAD

**Issue by mwaurahk** [opencdms/Climsoft#523]
HI all,
I have a Client - Server  setup of Climsoftv4 in KMD and we have a number of Key Entry staff working at the same time on a particular form. The issue is that when one saves and clears the form with Add new it doesn't happen..instead it displays data for a different station being worked on by the other Key Entry staff. So one is now forced to key in the station no, date he/she's working on - all the time they click Add new for the form to clear. Is it possible to maintain the station and move to the following day or month without having to key in the station and date again?

**Comment by smachua** [comment]
This issue was earlier raised as #453 and addressed to @Patowhiz  and @maxwellfundi. I hope it will be resolved soon.

**Comment by shadrackkibet** [comment]
I think this issue has been fixed.

**Comment by mhabimana** [comment]
@mwaurahk- Can you please test this and let us know before this is closed? On my side it is working well.

**Comment by Patowhiz** [comment]
Yes

**Comment by shadrackkibet** [comment]
So far the issue that I observed with three clients entering the data for the same station is that sometimes the form displays the data that is being entered by another user when they add new. This forces the data entry clerk to adjust the dates which is very annoying. Imagine a case when you have more clients working at the same time. This is a separate issue from above.

**Comment by Patowhiz** [comment]
Yes, we have not yet implemented the double data entry feature in the new data entry forms

**Comment by mhabimana** [comment]
@patowhiz - This is not double entry but it is the process of entering data for the same station  but different period of time using different Clients (PCs). We need to discuss further how to solve this issue as pointed out by @shadrackkibet

**Comment by Patowhiz** [comment]
Oh yes. I think this problem is related to the sequencer. We'll be testing the new implementation to see if this is solved.
